MIGRATION.md

```diff
@@ -2,290 +2,55 @@
-
-###### V2 command
-```terminal
-newman --url https://a.com/collection.json --environment-url https://a.com/env.json --noColor --outputFile jsonOut.json --testReportFile xmlOut.xml --html htmlOutput.html --outputFileVerbose verboseOut.log
-```
-###### V3 equivalent
-```terminal
-newman run https://a.com/collection.json --environment https://a.com/env.json --reporters cli,html,json,junit --reporter-json-export jsonOut.json --reporter-junit-export xmlOut.xml --reporter-html-export htmlOutput.html
+### Using HTML Reporter
+The inbuilt HTML reporter has been moved to a standalone reporter. Install it with:
+```console
+$ npm install -g newman-reporter-html
 ```
+Installation should be global if newman is installed globally, local otherwise. (Remove `-g` flag from the above command for a local installation.)
 
-#### CLI migration table
-| V2 CLI option | V3 equivalent | V2 option Status |
-|---------------|---------------|------------------|
-| -c --collection | N.A | Deprecated (Pass the collection file path without the collection flag) |
-| -u --url | N.A | Deprecated (Pass the collection file URL without the collection flag) |
-| --environment-url | N.A | Deprecated (Pass the environment file URL to -e --environment) |
-| -f --folder | --folder | Deprecated. The V3 folder option has been reduced to --folder |
-| -d --data | -d --iteration-data | Deprecated |
-| -n --number | -n --iteration-count | Deprecated |
-| -i --import | N.A | Deprecated |
-| -p --pretty | N.A | Discontinued |
-| -G --exportGlobals | --export-globals | Deprecated |
-| -E --exportEnvironment | --export-environment | Deprecated |
-| -y --delay | --delay-request | Deprecated |
-| -r --requestTimeout | --timeout-request | Deprecated |
-| -s --stopOnError | --bail | Deprecated |
-| -j --noSummary | --reporter-cli-no-summary | Deprecated |
-| -C --noColour | --no-color | Deprecated |
-| -S --noTestSymbols | --disable-unicode | **Discontinued** |
-| -l --tls | N.A | **Discontinued** |
-| -x --exitCode | -x --suppress-exit-code | Deprecated (The V3 option takes no arguments, and forces an exit code of 0)|
-| -w --whiteScreen | N.A | **Discontinued** |
-| -o --outputFile | --reporter-json-export | Deprecated |
-| -t --testReportFile | --reporter-junit-export | Deprecated |
-| -H --html | --reporter-html-export | Deprecated |
-| -O --outputFileVerbose | N.A | **Discontinued** |
-| -R --avoidRedirects | --ignore-redirects | Deprecated |
+The complete installation and usage guide is available here: https://github.com/postmanlabs/newman-reporter-html
 
-### Library usage
+### CSV auto parse
+A [fix][pr1609] has been made to avoid parsing numbers inside quotes.<br/>
+Example, `"000123"` will not be parsed to `123` like before.
 
-#### Sample use cases
+Fixes issues: [#1100][i1100], [#1215][i1215] & [#1346][i1346]
 
-##### 1. A collection run with all basic options configured, excluding reporter actions
+### Default timeouts
+The default timeout values are changed from 0 to infinity.<br/>
```

Comment:
Specify the PR link in parentheses after the sentence, not in a separate statement below. Also, the default timeouts were not Infinity before, it was 3 minutes for the global timeout.